_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -743,12 +731,6 @@ public function update_data( $request ) {
 					$updated = get_option( $option ) != $value ? update_option( $option, (bool) $value ? 'letitsnow' : '' ) : true;
 					break;
 
-				case 'akismet_show_user_comments_approved':
```

Comment:
We can't remove this. This is for the checkbox in **Spam filtering** settings card